vendor/init.go

```diff
@@ -2,28 +2,45 @@
 package main
 
 import (
+	"context"
+	"crypto/md5"
 	"crypto/sha256"
 	"encoding/hex"
 	"flag"
 	"fmt"
+	"hash"
 	"io"
+	"io/ioutil"
 	"net/http"
 	"os"
 	"os/exec"
 	"path"
+	"strings"
 
+	"cloud.google.com/go/storage"
 	"github.com/golang/glog"
+	"google.golang.org/api/option"
+)
+
+// consts for downloading Google chrome browser.
+const (
```

Comment:
Unlike in other languages, constants in Go do not need to be first in the file. Move these to the lowest scope possible, just before they are used.